src/main/cms/cms.c

```diff
@@ -367,6 +377,7 @@ static int cmsDrawMenuItemValue(displayPort_t *pDisplay, char *buff, uint8_t row
 #else
     colpos = smallScreen ? rightMenuColumn - maxSize : rightMenuColumn;
 #endif
+    cmsSanitizeBuffer(buff);
```

Comment:
Looking at this, it will be a bit cleaner if `cmsSanitizeBuffer()` is called from within `displayWrite()`.

Reply:
`displayWrite()` is also used for OSD isn't it?  might have a pretty catastrophic effect there.

Reply:
osd/osd.c:    displayWrite(osdDisplayPort, 20, 6, DISPLAYPORT_ATTR_NONE, string_buffer);
osd/osd.c:    displayWrite(osdDisplayPort, 7, 8,  DISPLAYPORT_ATTR_NONE, CMS_STARTUP_HELP_TEXT1);
osd/osd.c:    displayWrite(osdDisplayPort, 11, 9, DISPLAYPORT_ATTR_NONE, CMS_STARTUP_HELP_TEXT2);
osd/osd.c:    displayWrite(osdDisplayPort, 11, 10, DISPLAYPORT_ATTR_NONE, CMS_STARTUP_HELP_TEXT3);
osd/osd.c:        displayWrite(osdDisplayPort, 5, 12, DISPLAYPORT_ATTR_NONE, dateTimeBuffer);
osd/osd.c:    displayWrite(osdDisplayPort, 2, y, DISPLAYPORT_ATTR_NONE, text);
osd/osd.c:    displayWrite(osdDisplayPort, 20, y, DISPLAYPORT_ATTR_NONE, ":");
osd/osd.c:    displayWrite(osdDisplayPort, 22, y, DISPLAYPORT_ATTR_NONE, value);
osd/osd.c:        displayWrite(osdDisplayPort, 2, displayRow, DISPLAYPORT_ATTR_NONE, buff);
osd/osd.c:        displayWrite(osdDisplayPort, 2, top++, DISPLAYPORT_ATTR_NONE, "  --- STATS ---");
osd/osd.c:    displayWrite(osdDisplayPort, 12, 7, DISPLAYPORT_ATTR_NONE, "ARMED");
osd/osd_elements.c:    return displayWrite(element->osdDisplayPort, x, y, attr, s);

Reply:
Lol, not sure how I overlooked this - we could wrap this into a `cmsDisplayWrite()` function that does the sanitising.

Reply:
fixed